games-puzzle/kpat/kpat-17.12.1.recipe

```diff
@@ -0,0 +1,139 @@
+CHECKSUM_SHA256="51f42d0c7302d9fd42fa3dffec2ea7a6e13df6408ecabaabdf77813c264a3142"
+SOURCE_FILENAME="kpat-$portVersion.tar.gz"
+PATCHES="kpat-$portVersion.patchset"
+ADDITIONAL_FILES="kpat.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	kpat$secondaryArchSuffix = $portVersion
+	app:kpat$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libKF5Attica$secondaryArchSuffix
+	lib:libKF5Auth$secondaryArchSuffix
+	lib:libKF5Bookmarks$secondaryArchSuffix
+	lib:libKF5Codecs$secondaryArchSuffix
+	lib:libKF5Completion$secondaryArchSuffix
+	lib:libKF5ConfigCore$secondaryArchSuffix
+	lib:libKF5ConfigWidgets$secondaryArchSuffix
+	lib:libKF5CoreAddons$secondaryArchSuffix
+	lib:libKF5Crash$secondaryArchSuffix
+	lib:libKF5DBusAddons$secondaryArchSuffix
+	lib:libKF5GuiAddons$secondaryArchSuffix
+	lib:libKF5I18n$secondaryArchSuffix
+	lib:libKF5ItemViews$secondaryArchSuffix
+	lib:libKF5JobWidgets$secondaryArchSuffix
+	lib:libKF5KDEGames$secondaryArchSuffix
+	lib:libKF5KIOCore$secondaryArchSuffix
+	lib:libKF5NewStuff$secondaryArchSuffix
+	lib:libKF5Service$secondaryArchSuffix
+	lib:libKF5WidgetsAddons$secondaryArchSuffix
+	lib:libKF5XmlGui$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Qml$secondaryArchSuffix
+	lib:libQt5Quick$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	extra_cmake_modules >= 5.39
+	haiku${secondaryArchSuffix}_devel
+	devel:libKF5Attica$secondaryArchSuffix
+	devel:libKF5Auth$secondaryArchSuffix
+	devel:libKF5Bookmarks$secondaryArchSuffix
+	devel:libKF5Codecs$secondaryArchSuffix
+	devel:libKF5Completion$secondaryArchSuffix
+	devel:libKF5ConfigCore$secondaryArchSuffix
+	devel:libKF5ConfigWidgets$secondaryArchSuffix
+	devel:libKF5CoreAddons$secondaryArchSuffix
+	devel:libKF5Crash$secondaryArchSuffix
+	devel:libKF5DBusAddons$secondaryArchSuffix
+	devel:libKF5GuiAddons$secondaryArchSuffix
+	devel:libKF5I18n$secondaryArchSuffix
+	devel:libKF5ItemViews$secondaryArchSuffix
+	devel:libKF5JobWidgets$secondaryArchSuffix
+	devel:libKF5KDEGames$secondaryArchSuffix
+	devel:libKF5KIOCore$secondaryArchSuffix
+	devel:libKF5NewStuff$secondaryArchSuffix
+	devel:libKF5Service$secondaryArchSuffix
+	devel:libKF5Solid$secondaryArchSuffix
+	devel:libKF5WidgetsAddons$secondaryArchSuffix
+	devel:libKF5XmlGui$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Qml$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:gettext$secondaryArchSuffix
+	cmd:make
+	cmd:python2
+	cmd:update_mime_database$secondaryArchSuffix
+	"
+
+BUILD()
+{
+
+	mkdir -p build
+	cd build
+
+	cmake .. \
+		-DECM_DIR=/system/data/cmake/Modules/ECM/cmake \
+		-DCMAKE_INSTALL_PREFIX=$prefix
```

Comment:
```diff
-		-DCMAKE_INSTALL_PREFIX=$prefix
+		-DCMAKE_INSTALL_PREFIX=$prefix \

Reply:
This is one is not necessary as /boot/system is already the default in Cmake.

Reply:
Sorry, maybe an unified diff would let you see why

```diff
 
 	cmake .. \
 		-DECM_DIR=/system/data/cmake/Modules/ECM/cmake \
-		-DCMAKE_INSTALL_PREFIX=$prefix
+		-DCMAKE_INSTALL_PREFIX=$prefix \
 		-DKDE_INSTALL_BINDIR=$appsDir/KPatience \
 		-DKDE_INSTALL_LIBDIR=$appsDir/KPatience/lib \
 		-DKDE_INSTALL_DATADIR=$dataDir \
```